docs/tiger_data_spec.md

```diff
@@ -133,3 +136,113 @@ A typical search will return records in the following format when using ElasticS
 
 }
 ```
+
+## Data creation
+
+The census geocoder uses Elasticsearch synonyms to resolve abbreviations (i.e. St for Street, or MD for Maryland).
+The synonyms.txt file with the synonyms definition must be installed in every node in the cluster, in the same directory as the elasticsearch.yml configuration file.
+
+* First, create the index, with the synonyms analyzer settings, and apply that analyzer to the corresponding fields:
```

Comment:
The intention is to document reproducible steps for someone starting from scratch. How to add synonyms is out of scope, maybe a link to Elasticsearch documentation would suffice. 

As for synonyms being applied to address points, it should work but we should keep an eye on that because the search is done in a completely different way. I was planning on optimizing that separately, but go ahead and include it